.github/CODEOWNERS

```diff
@@ -812,6 +812,7 @@ embed.go @grafana/grafana-as-code
 /.github/workflows/trivy-scan.yml @grafana/grafana-backend-services-squad
 /.github/workflows/changelog.yml @zserge
 /.github/workflows/actions/changelog @zserge
+/.github/workflows/frontend-unit-tests.yml @grafana/grafana-frontend-platform
```

Comment:
Not @\grafana/grafana-developer-enablement-squad?

Reply:
no we don't own workflows for testing other people's code, further maintenance is their responsibility.